tests/queries/0_stateless/02967_mysql_settings_override.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+# Tag no-fasttest: requires mysql client
+
+# Tests the override of certain MySQL proprietary settings to ClickHouse native settings
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+CHANGED_SETTINGS_QUERY="SELECT name, value FROM system.settings WHERE name IN ('send_timeout', 'receive_timeout') AND changed;"
+
+TEST_TABLE="mysql_settings_override_test"
+
+DROP_TABLE="DROP TABLE IF EXISTS $TEST_TABLE;"
+CREATE_TABLE="CREATE TABLE $TEST_TABLE (s String) ENGINE MergeTree ORDER BY s;"
+INSERT_STMT="INSERT INTO $TEST_TABLE VALUES ('a'), ('b'), ('c'), ('d');"
+SELECT_STMT="SELECT * FROM $TEST_TABLE;"
```

Comment:
Better put in an `ORDER BY s` at the end of the `SELECT` to guarantee a stable result order.